actionpack/lib/action_dispatch/middleware/host_authorization.rb

```diff
@@ -34,6 +34,10 @@ def allows?(host)
         end
       end
 
+      def block?(host)
+        !allows?(host)
```

Comment:
I thought I can rewrite `#allows?` using `@hosts.none?` and remove `#allows?` completely since we don't use it but `#none?` doesn't read as clear for me as `#any?`.